time/chrono.h

```diff
@@ -65,8 +65,8 @@ inline EPOCH_MILLISECONDS Now() {
 }
 
 #else
+#warning 'thread_local' is not supported by compiler
```

Comment:
I tend to keep it here:
* otherwise `make all` from top-level directory will generate a bunch of similar warnings
* it's informative here as a sign of possible problems in particular implementation